### PR TITLE
Detailed instructions for running this sample app

### PR DESCRIPTION
**Pull request by desokroshan**

### CODE REVIEW

File: AmazonCognitoAuthDemo/README.md [hunk 1]
```diff
@@ -1,7 +1,36 @@
...
-Make changes to the values found in the [strings.xml file](app/src/main/res/values/strings.xml)
-See following sections for explanation on what these fields are.
+1. Create Your user pool on the Cognito console
+   - Follow the steps outlined [here](https://github.com/awslabs/aws-sdk-android-samples/blob/master/AmazonCognitoYourUserPoolsDemo/README.md#running-amazoncognitoyouruserpoolsdemo) to create a user pool.
+   - Configure the App Client as follows:
+      - Choose _App client settings_ from the navigation bar on the left-side of the console page.
+      - Select Cognito User Pool as one of the Enabled Identity Providers.
+      - Type a callback URL for the Amazon Cognito authorization server to call after users are authenticated. For the sample app set it to _myapp://_
+      - Set Sign out URL(s) to _myapp://_
+      - You can enable both the Authorization code grant and the Implicit code grant under Allowed OAuth flows.
```
Comment (minbi) [inline AmazonCognitoAuthDemo/README.md]: I would bold the words that match the console wording. Otherwise the capitalization is a little disorienting.

Comment (desokroshan) [reply AmazonCognitoAuthDemo/README.md]: Fixed

File: AmazonCognitoAuthDemo/README.md [hunk 2]
```diff
@@ -1,7 +1,36 @@
 # Running this sample
 
-Make changes to the values found in the [strings.xml file](app/src/main/res/values/strings.xml)
-See following sections for explanation on what these fields are.
+1. Create Your user pool on the Cognito console
```
[the remark this answers is not in the document]
Comment (desokroshan) [reply AmazonCognitoAuthDemo/README.md]: Fixed

File: AmazonCognitoAuthDemo/README.md [hunk 3]
```diff
@@ -1,7 +1,36 @@
...
+   - Configure a user pool domain
+      - On the Domain name page, type a domain prefix that's available.
+      - Make a note of the complete domain address.
+      - Choose Save changes.
+
+2. Download and import the AmazonCognitoYourUserPoolsDemo project into your Android Studio
+   - From the Welcome screen, click on "_Import project_".
+   - Browse to the AAmazonCognitoAuthDemo directory and click OK.
+   - Accept requests to add Gradle to the project.
+   - If the SDK reports missing Android SDK packages (such as Build Tools or the Android API package), import AWS SDK.
```
[the remark this answers is not in the document]
Comment (desokroshan) [reply AmazonCognitoAuthDemo/README.md]: Fixed

File: AmazonCognitoAuthDemo/README.md [hunk 4]
```diff
@@ -1,7 +1,36 @@
...
+      - Unless you specifically want to exclude one, select the check boxes for all of the Allowed OAuth scopes.
+      - Choose Save changes.
+   - Configure a user pool domain
+      - On the Domain name page, type a domain prefix that's available.
+      - Make a note of the complete domain address.
+      - Choose Save changes.
+
+2. Download and import the AmazonCognitoYourUserPoolsDemo project into your Android Studio
+   - From the Welcome screen, click on "_Import project_".
+   - Browse to the AAmazonCognitoAuthDemo directory and click OK.
```
[the remark this answers is not in the document]
Comment (desokroshan) [reply AmazonCognitoAuthDemo/README.md]: Fixed

**Pull request merged**